Gemfile

```diff
@@ -62,6 +62,12 @@ gem "sidekiq"
 # Preview email in browser
 gem "letter_opener", group: :development
 
+# yanked v0.3.5
+gem 'mimemagic', github: 'mimemagicrb/mimemagic', ref: '01f92d86d15d85cfd0f20dabd025dcbd36a8a60f'
```

Comment:
Do we need `mimemagic` gem? Rails is already upgraded to `6.1.3.2`.

Reply:
@akhil-gautam Good catch. It's not needed. 

I'm closing this PR either ways, since it's not needed at the moment.